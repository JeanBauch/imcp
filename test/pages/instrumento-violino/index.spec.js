import { mount } from '@vue/test-utils';

import Index from '~/pages/instrumento-violino/index.vue';

describe('Pages:instrumento-violino:index', () => {
	let mountConfig;

	beforeEach(() => {
		mountConfig = {
			stubs: {
				Breadcrumb: true,
				TitlePage: true,
				BarCursos: true,
				CardVideo: true,
				Form: true,
				CardDescription: true,
			},
		};
	});

	describe('Snapshots', () => {
		it('Should render all child componets', () => {
			const wrapper = mount(Index, mountConfig);
			expect(wrapper.html()).toMatchSnapshot();
		});
	});
});
