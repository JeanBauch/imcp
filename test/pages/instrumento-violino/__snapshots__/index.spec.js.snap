// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:instrumento-violino:index Snapshots Should render all child componets 1`] = `
"<div class=\\"bg-background\\">
  <section class=\\"container mx-auto px-6 sm:px-9 lg:px-24\\">
    <div class=\\"flex flex-1 py-3\\">
      <breadcrumb-stub crumbs=\\"[object Object],[object Object],[object Object]\\"></breadcrumb-stub>
    </div>
    <div class=\\"flex flex-1 items-center justify-center mt-4 md:mt-6\\">
      <titlepage-stub title=\\"Violino\\" titlecolor=\\"orange\\" separetorcolor=\\"yellow\\" width=\\"w-full\\"></titlepage-stub>
    </div>
  </section>
  <div class=\\"h-[120px] md:h-[500px] bg-cover my-6 md:my-8 \\" style=\\"background-image: url(https://images.unsplash.com/photo-1612225330812-01a9c6b355ec?ixlib=rb-1.2.1&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=2070&amp;q=80);\\">
    <div class=\\"hidden md:flex mx-auto  justify-end container h-[500px] items-end px-6 sm:px-9 lg:px-24\\"><img src=\\"/icons/medium-colors/violincurso.svg\\" alt=\\"\\" srcset=\\"\\" class=\\"max-h-[160px] mb-4\\"></div>
  </div>
  <main class=\\"mt-6 mb-56\\">
    <div class=\\"container mx-auto px-6 sm:px-9 lg:px-24\\">
      <div class=\\"flex flex-col justify-center gap-6 md:gap-10 bg-white md:px-14 md:pb-4 md:pt-14 rounded-3xl border border-primary-beige-300\\">
        <div class=\\"flex flex-col-reverse items-center xl:flex-row gap-6 md:gap-20\\">
          <div class=\\"text-base md:text-lg xl:max-w-[592px] text-black leading-5 md:leading-6\\">
            <h2 class=\\"font-semibold text-3xl md:text-[40px] text-primary-beige-700 pb-3 md:pb-6\\">
              Aulas de Violino
            </h2>
            <p>
              Qualquer menino ou menina com vocação poderá ingressar no Coral
              dos Canarinhos de Petrópolis e também no Coral das Meninas dos
              Canarinhos de Petrópolis por meio do Curso de Aprendizes
              oferecido, gratuitamente, pelo Instituto dos Meninos Cantores de
              Petrópolis, para crianças do 3º ano das redes pública e particular
              de ensino.<br class=\\"md:hidden\\"><br> O nosso objetivo vai além da formação musical, formando
              cidadãos de bem.
            </p>
            <div class=\\"block\\">
              <h3 class=\\"font-semibold text-2xl mb-2 mt-4 md:mt-4 md:mb-4\\">
                Aulas:
              </h3>
              <ul class=\\"text-lg\\">
                <li><i class=\\"text-primary-beige-700 text-base md:text-2xl mr-2\\">●</i> Iniciação musical e flauta doce</li>
                <li class=\\"my-1\\"><i class=\\"text-primary-beige-700  text-base  md:text-2xl mr-2\\">●</i> Teoria musical e solfejo
                </li>
                <li><i class=\\"text-primary-beige-700 text-base  md:text-2xl mr-2\\">●</i> Técnica Vocal</li>
              </ul>
            </div> <button class=\\"w-[155px] md:w-[250px] h-9 md:h-12 rounded-3xl font-semibold text-base md:text-lg text-white hover:brightness-105 transition-all cursor-pointer bg-primary-beige-700 my-8\\">
              TENHO INTERESSE
            </button>
          </div>
          <div class=\\"hidden md:block w-80 xl:w-4/5 h-60 lg:h-60 xl:h-96 2xl:h-[460px] bg-cover bg-center bg-[url('https://www.bomjesus.br/galeria/getImage/140/447740909258037.jpg')]\\" style=\\"border-radius: 60% 40% 33% 67% / 36% 51% 49% 64%;\\"></div>
        </div>
      </div>
    </div>
    <div class=\\"my-10 container mx-auto px-6 sm:px-9 lg:px-24\\">
      <div class=\\"bg-white grid grid-cols-1 lg:grid-cols-2 gap-20 p-6 md:p-14 justify-items-center rounded-[30px] border border-primary-beige-300\\">
        <div class=\\"w-full h-[425px] rounded-[32px] hidden md:block\\" style=\\"background: url(https://images.unsplash.com/photo-1603741209938-50fc882da856?ixlib=rb-1.2.1&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=1170&amp;q=80);\\"></div>
        <div class=\\"flex flex-col justify-center\\">
          <h1 class=\\"text-3xl md:text-[40px] font-semibold mb-10\\">
            Professores <span class=\\"text-primary-beige-500\\">Responsáveis</span></h1>
          <div class=\\"flex items-center justify-left mb-8 text-2xl\\"><i class=\\"w-18 h-18 min-w-[62px] md:w-20 md:h-20 md:min-w-[80px] rounded-full mr-4 lg:mr-8\\"></i>
            <div class=\\"flex flex-col justify-center \\"><span class=\\"font-medium md:font-normal text-xl md:text-2xl\\">Profº Marco Aurélio Lischt</span> <a href=\\"#\\" class=\\"underline flex gap-8 items-center text-base md:text-2xl text-primary-beige-500\\">VER CURRÍCULO <i><svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"18\\" height=\\"18.828\\" viewBox=\\"0 0 18 18.828\\" class=\\"hover:translate-x-4 transition-all\\">
                    <g id=\\"Icon_feather-arrow-right\\" data-name=\\"Icon feather-arrow-right\\" transform=\\"translate(1 1.414)\\">
                      <path id=\\"Caminho_1292\\" data-name=\\"Caminho 1292\\" d=\\"M7.5,18h16\\" transform=\\"translate(-7.5 -10)\\" fill=\\"none\\" stroke=\\"#DEB076\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"2\\"></path>
                      <path id=\\"Caminho_1293\\" data-name=\\"Caminho 1293\\" d=\\"M18,7.5l8.4,8-8.4,8\\" transform=\\"translate(-10.395 -7.5)\\" fill=\\"none\\" stroke=\\"#DEB076\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"2\\"></path>
                    </g>
                  </svg></i></a></div>
          </div>
          <div class=\\"flex items-center justify-left mb-8 text-2xl\\"><i class=\\"w-18 h-18 min-w-[62px] md:w-20 md:h-20 md:min-w-[80px] rounded-full mr-4 lg:mr-8\\"></i>
            <div class=\\"flex flex-col justify-center \\"><span class=\\"font-medium md:font-normal text-xl md:text-2xl\\">Profª Elza Pereira</span> <a href=\\"#\\" class=\\"underline flex gap-8 items-center text-base md:text-2xl text-primary-beige-500\\">VER CURRÍCULO <i><svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"18\\" height=\\"18.828\\" viewBox=\\"0 0 18 18.828\\" class=\\"hover:translate-x-4 transition-all\\">
                    <g id=\\"Icon_feather-arrow-right\\" data-name=\\"Icon feather-arrow-right\\" transform=\\"translate(1 1.414)\\">
                      <path id=\\"Caminho_1292\\" data-name=\\"Caminho 1292\\" d=\\"M7.5,18h16\\" transform=\\"translate(-7.5 -10)\\" fill=\\"none\\" stroke=\\"#DEB076\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"2\\"></path>
                      <path id=\\"Caminho_1293\\" data-name=\\"Caminho 1293\\" d=\\"M18,7.5l8.4,8-8.4,8\\" transform=\\"translate(-10.395 -7.5)\\" fill=\\"none\\" stroke=\\"#DEB076\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"2\\"></path>
                    </g>
                  </svg></i></a></div>
          </div>
        </div>
      </div>
    </div>
    <div class=\\"bg-primary-beige-300 relative\\">
      <div class=\\"container flex flex-row flex-wrap justify-center 2xl:justify-between mx-auto pt-10 px-6 sm:px-9 lg:px-24\\">
        <carddescription-stub bgcolor=\\"beige\\" bgcolorweight=\\"700\\" textcolor=\\"bordo\\" complementbordercolor=\\"bordo\\" instrumento=\\"Violino\\" disponivel=\\"true\\" class=\\"z-20\\"></carddescription-stub>
        <form-stub color=\\"bordo\\" bgcolorweight=\\"700\\" text=\\"true\\" tel=\\"true\\" telcolor=\\"#991C3A\\" inputbg=\\"beige\\" labelweight=\\"normal\\" btncolor=\\"beige\\" bordercolor=\\"beige\\" width=\\"w-full\\" class=\\"z-20\\"></form-stub>
      </div> <img src=\\"complement/bg-instrumento.svg\\" alt=\\"\\" srcset=\\"\\" class=\\"absolute bottom-0 w-full\\">
    </div>
    <div class=\\"mx-auto container px-6 sm:px-9 lg:px-24\\">
      <cardvideo-stub color=\\"orange\\"></cardvideo-stub>
      <barcursos-stub></barcursos-stub>
    </div>
  </main>
</div>"
`;
